lib/views/screens/home_page.dart

```diff
@@ -18,6 +18,84 @@ class _HomePageState extends State<HomePage> {
       "https://tiles.stadiamaps.com/tiles/alidade_smooth_dark/{z}/{x}/{y}{r}.png";
   final apiKey = "e2421755-eb2a-4461-b7fe-29a8a7b5aaec";
 
+  Map<String, List> blockCoordinates = {
```

Comment:
can specify the type of List further, like List<List<List<double>>>>. 
this will eliminate the redundant `as double` below.

Reply:
The list is not nested for all blocks so I can't specify the format, but as the coords are Doubles anyway, we won't need the typecasting, so I just removed it